.github/workflows/CI.yml

```diff
@@ -16,11 +16,24 @@ jobs:
         os: [windows-latest, macOS-latest]
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
     - name: Use Node.js ${{ matrix.node_version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node_version }}
+        cache: 'npm'
+    - name: Cache Prettier
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-prettier
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: node_modules/.cache/prettier
+        key: ${{ runner.os }}-prettier-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-prettier-${{ env.cache-name }}-
+          ${{ runner.os }}-prettier-
+          ${{ runner.os }}-
```

Comment:
I think you have to move this after the `Install Dependencies` step since `npm ci` nukes `node_modules`.